example_rails_app/spec/models/user_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
```

Comment:
Realised this PR is a bit hard to parse with the mock generation and hound dog comments.
The main gist of the PR is from this point down. `example_rails_app/spec/models/user_spec.rb` and `example_rails_app/spec/rails_helper.rb`